### PR TITLE
Shellcheck failures print-workspace-status.sh, run-in-gopath.sh

### PR DESCRIPTION
**Pull request by thesattiraju**
Signed-off-by: ds-ms <desattir@microsoft.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**: 
#72956

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:
```release-note
NONE
```

### CODE REVIEW

File: hack/print-workspace-status.sh [hunk 1]
```diff
@@ -20,7 +20,8 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-export KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+export KUBE_ROOT
```
Comment (ixdy) [inline hack/print-workspace-status.sh]: I'm not sure we need to export this at all.

**Comment by ixdy** [comment]
Please squash your commits. Also please remove the "Fixes ..." line from your PR description, since that will close the referenced issue. Thanks!

/ok-to-test

File: hack/print-workspace-status.sh (hunk 1 above)
Comment (ds-ms) [reply hack/print-workspace-status.sh]: Removed it.

**Comment by ds-ms** [comment]
@ixdy does this PR need any sig's approval?

**Comment by ixdy** [comment]
apparently it does.
/sig testing release

**Pull request merged**